examples/example-app/build.sbt

```diff
@@ -0,0 +1,35 @@
+import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
+import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
+import org.virtuslab.ash.AkkaSerializationHelperPlugin
+
+name := "example-app"
+version := "0.1"
+scalaVersion := "2.13.6"
+
+val circeVersion = "0.14.2"
+val akkaVersion = "2.6.19"
+val logbackVersion = "1.2.11"
+
+lazy val `example-app` = project
+  .in (file("."))
+  .enablePlugins(AkkaSerializationHelperPlugin)
+  .settings(multiJvmSettings: _*)
+  .settings(
+    libraryDependencies ++= akkaDependencies ++ ashDependencies ++ Seq(logbackDependency, circeDependency),
+    fork :=  true,
+    Global / cancelable := false
```

Comment:
Hmmm what's the effect of this setting here?

Reply:
I don't know to be honest. I took this setting from https://github.com/akka/akka-samples/blob/2.6/akka-sample-cluster-scala/build.sbt#L24 - and works fine.

Reply:
Okay, no need to investigate then, I thought it's your own addition